rasa/shared/utils/io.py

```diff
@@ -306,7 +306,7 @@ def replace_environment_variables() -> None:
     env_var_pattern = re.compile(r"^(.*)\$\{(.*)\}(.*)$")
     yaml.Resolver.add_implicit_resolver("!env_var", env_var_pattern, None)
 
-    def env_var_constructor(loader, node):
+    def env_var_constructor(loader: BaseConstructor, node: Any) -> AnyStr:
```

Comment:
what's a `AnyStr` and why is it not a pure `Text`?

Reply:
PyCharm suggestion - seems to think it can return either `str` or `bytes`: https://docs.python.org/3/library/typing.html#typing.AnyStr

Reply:
Thanks for the link! In our case it in our case `expandvars` will always return `str`, won't it?